v63004/u_inref/gtm8791.exp

```diff
@@ -0,0 +1,26 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+set timeout 60
+spawn /usr/local/bin/tcsh -f
+set cenable(0) "CENABLE"
+set cenable(1) "NOCENABLE"
+
+send -- "\$gtm_dist/lke\r"
+
+expect -exact "LKE>"
+# Send <Ctrl-Z>
+send -- "\x1A\r"
+
+expect -exact "Suspended (signal)"
+send -- "exit\r"
```

Comment:
The exit will exit the shell. But the backgrounded LKE is still running and needs to be terminated. We don't want the test to leave any processes behind.